Tom small things

tblah opened the pull request:
rnn branch + small sections

tblah committed:
Merge branch 'george' into rnn

Hopefully avoiding conflicts down the line.

tblah committed:
First Models section on RNN

More to come in the grid search section

tblah committed:
Mention RNN tests with stemming and stop-word removal

Also emacs insisted on limiting line length. I find it easier to read anyway

tblah committed:
RNN grid search extension

Needs some proof reading and a final result to be added but it is nearly there

tblah committed:
Proof reading of RNN stuff

tblah committed:
Merge remote-tracking branch 'origin/master' into rnn

tblah committed:
Many small sections

Contributions to intro
Random Forest
SVM
Ensemble
Future work (please add to this?)

tblah committed:
Final results

tblah committed:
Merge branch 'rnn' into tom-small-things

Adimote committed:
initial paragraph of word2vec

tblah committed:
Mention new and improved Random Forest

Adimote committed:
mention Word2Vec and people are compared

Adimote committed:
oxford comma

tblah committed:
Probabilisitic SVM and Forrest

tblah committed:
Fix small typo "encodings in a large vector-spaces"

tblah committed:
Remove now redundant future work

tblah committed:
Merge branch 'word2vec' into tom-small-things

tblah committed:
Sentence Transformations section

tblah committed:
Confusion matrices for SVM and Forest

tblah committed:
Merge remote-tracking branch 'origin/master' into tom-small-things

tblah committed:
Merge remote-tracking branch 'origin/rnn' into tom-small-things

tblah committed:
Future work regarding probabilistic classifier

Adimote commented on first_models.tex:
```diff
@@ -53,9 +53,35 @@ \subsection{Bag of Words}
+Based on impressive successes in software author recognition using random forest
+methods \cite{c3}, there was interest in the accuracy of a random forest
+predictor, even with its unsuitability for use in a competition judged by
+log-loss. Initially the forest was fed encoded sentences directly.
```
is it worth explaining why log-loss is bad for random forest? I'm more asking this because I myself don't know why it's bad.

Adimote commented on future_work.tex:
```diff
@@ -1,2 +1,13 @@
\ No newline at end of file
+\label{sec:future_work}
+
+The ensemble could be expended to include more classifiers and some additional
```
"expended" should be "expanded"

Adimote commented on future_work.tex:
```diff
@@ -1,2 +1,13 @@
+At the moment the probabilistic model assumes that the probabilities that words
+occur in a sentence are independent of each-other. This is clearly not the case.
+One way to extend the model would be to consider closer approximations to the
+true distribution for example by considering pairs of words instead of words on
```
distribution,  <-comma

tblah committed:
Andy's feedback on pull RNN pull request

s/affect/effect

tblah committed:
Merge branch 'rnn' into tom-small-things

tblah committed:
Andy's comments

first_models.tex - explain why a random forest is un-suitable for log-loss

future_work.tex - s/expended/expanded, added missing comma

Also added biber output files to the gitignore
